db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql

```diff
@@ -17,6 +22,4 @@ update keymgr.key_policy_def set pre_expire_days=60, access_allowed='NA' where a
 update keymgr.key_policy_def set pre_expire_days=60, access_allowed='NA' where app_id='KERNEL';
 update keymgr.key_policy_def set pre_expire_days=1125, access_allowed='NA' where app_id='ROOT';
 update keymgr.key_policy_def set pre_expire_days=30, access_allowed='NA' where app_id='BASE';
-update keymgr.key_policy_def set pre_expire_days=395, access_allowed='NA' where app_id='PMS';
```

Comment:
Why is PMS removed from here ?

Reply:
It is not removed, because of the new line it moved below.

Reply:
oh ok